pnpm-lock.yaml

```diff
@@ -12473,6 +12476,8 @@ snapshots:
 
   ilib@14.20.0: {}
 
+  ilib@14.21.0: {}
```

Comment:
The pnpm-lock.yml file is an automatically generated file, but I'm curious about why it is being added instead of being overwritten.